scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -1401,6 +1404,40 @@ def _consolidated_checkpoint_file_path(self):
             CONSOLIDATED_CHECKPOINTS_FILE_NAME,
         )
 
+    def __process_checkpoints_on_startup(self, checkpoints):
+        # type: (dict) -> dict
+        """
+        Process checkpoints on startup and ignore any entries which should be ignored based on the
+        "ignore_checkpoints_on_startup_path_globs" config option value.
+
+        NOTE: This method should only be called once on startup before doing any other work (aka in
+        the run() method before consolidating the checkpoints and doing any other work).
+        """
+        checkpoints = checkpoints or {}
+        ignore_checkpoints_on_startup_path_globs = (
+            self.__config.ignore_checkpoints_on_startup_path_globs
+        )
+
+        def path_matches_any_glob(file_path):
+            for glob in ignore_checkpoints_on_startup_path_globs:
+                if fnmatch.fnmatch(file_path, glob):
+                    return True
+
+            return False
+
+        checkpoint_file_paths = list(checkpoints.keys())
+
+        for file_path in checkpoint_file_paths:
+            if path_matches_any_glob(file_path=file_path):
+                log.log(
+                    scalyr_logging.DEBUG_LEVEL_0,
```

Comment:
Right now this is being logged under INFO, but I'm kinda torn between DEBUG and INFO here.

INFO may be a bit verbose and some users may get concerned by that log message, but in case we change it to DEBUG, it may make debugging any issues related to this functionality harder.

Reply:
If I'm not mistaken, that has to be only on startup, so it shouldn't generate a constant noise if we keep it as INFO

Reply:
Correct, only once on startup (message per ignored log file), but it may still alert some users so dunno.

Reply:
Recommend keeping it at INFO.  The changelog .md is pretty clear.  Should we update the readme.md?  I could not locate any content on this.

Reply:
@dariocalia Thanks for the input. 

Not sure what exactly you mean with updating readme though - readme in this repo is mostly used for people interested in contributing to the project and / or using the source code of the agent directly.

So not sure what we should put in the readme with regards to the change / bug fix :) Or did you perhaps mean RELEASE_NOTES.md?